### PR TITLE
Sonobi Bid Adapter: add additional sizes to bid request

### PR DESCRIPTION
**Pull request by About7Sharks**
## Type of change

- [x] Refactoring (no functional changes, no api changes)

## Description of change

`_validateSize()` takes in a single parameter, "bid". The function takes all the possible sizes from the bid object and returns them as a string in the csv format. It accesses the different properties of the bid object to find the sizes, then parse it and throw it into a Set to uniquify it. Then it spreads it to an array and joins it into a csv of sizes.
The main purpose of this function is to check for any possible size fields within the bid object, and to parse and format that information into a single, consistent string in csv format which can be easily used by the rest of the program.

### CODE REVIEW

File: integrationExamples/gpt/hello_world.html [hunk 1]
```diff
@@ -1,91 +0,0 @@
-<!--
```
Comment (patmmccann) [inline integrationExamples/gpt/hello_world.html:1]: please restore

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: integrationExamples/gpt/hello_world.html (hunk 1 above)
Comment (Rothalack) [reply integrationExamples/gpt/hello_world.html]: Hey @About7Sharks , this file needs to be restored, not deleted.

Comment (Rothalack) [reply integrationExamples/gpt/hello_world.html]: I just realized that you did actually add it back, it's the files changed view on this PR that's misleading. I'll mark this resolved again.

**Comment by About7Sharks** [comment]
Awesome thanks! Apologies about that, I usually use `git add .` but when  testing using the `hello_world.html` It added those changes which I fudged the `git rm` command leading to the untracked file.

File: integrationExamples/gpt/hello_world.html (hunk 1 above)
Comment (About7Sharks) [reply integrationExamples/gpt/hello_world.html]: Awesome thanks! Apologies about that, I usually use `git add .` but when  testing using the `hello_world.html` It added those changes which I fudged the `git rm` command leading to the untracked file.

**Review by Rothalack — APPROVED** [review]
Working in testing for me, should be good to go.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**